generate_passwords.py

```diff
@@ -0,0 +1,36 @@
+import base64
+import os
+
+fn = ".pw"
+vn = {}
+
+pwvars = ["POSTGRES_PASSWORD", "CKAN_DB_PASSWORD", "DATASTORE_READONLY_PASSWORD","CKAN_SYSADMIN_PASSWORD"]
+
+print("[setup_passwords] attempting to setup secure passwords")
+
+with open(fn, 'w') as f:
+    f.truncate(0)
+    for pwvar in pwvars:
```

Comment:
Probably better to use https://docs.python.org/3/library/secrets.html#secrets.token_urlsafe  `secrets.token_urlsafe`, as `/dev/urandom` is somewhat platform dependent.